module/test/cardscounter.test.js

```diff
@@ -0,0 +1,17 @@
+const cardscounter = require('./cardscounter');
+
+test('should update the innerHTML of #cardNum with the correct number of cards', () => {
+  // Arrange
+  document.body.innerHTML = '<div id="cardNum"></div>';
+  const cards = [
+    { title: 'Card 1' },
+    { title: 'Card 2' },
+    { title: 'Card 3' },
+  ];
+
+  // Act
+  cardscounter(cards);
+
+  // Assert
+  expect(document.querySelector('#cardNum').innerHTML).toBe('Shows(3)');
+});
```

Comment:
- [ ] Please let' ensure that the `counter` function covers all the edge cases you can think about. (at least about the scenario in which there are 0 items on the page (counter function should return 0)🤓

- [ ] Anecdotally, it is important to test edges cases because this is where errors tend to arise 👍